scripts/init-database.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import yaml
+import sqlalchemy
+import os
+import getpass
+
+configs = 'configurations'
+path = 'root_path'
+
+
+def retreive_values():
+    global path
+    global configs
+    path= os.path.abspath(os.path.join(os.path.join(__file__,os.pardir),os.pardir))
+    stream = open(path+"/bassa.yml", "r")
+    configs = yaml.load(stream)
+
+
+def create_database():
+    try:
```

Comment:
Python won't throw if the asked ENV var is not set. you will simply get an empty string. Check for empty values.
![image](https://user-images.githubusercontent.com/1391468/34406234-e481d462-ebdd-11e7-8795-75dc3bd78077.png)